src/shell_audio.html

```diff
@@ -0,0 +1,1325 @@
+   transform="matrix(0.24 0 0 0.24 228.8599 216.5547)"
+   id="image155"></image><g
+   id="g157"><radialGradient
+   id="SVGID_9_"
+   cx="232.459"
+   cy="219.7041"
+   r="2.9995"
+   gradientTransform="matrix(1 0 0 0.75 0 54.926)"
+   gradientUnits="userSpaceOnUse"><stop
+   offset="0.0123"
+   style="stop-color:#FFFFFF"
+   id="stop160" /><stop
+   offset="0.4235"
+   style="stop-color:#FAFCF6"
+   id="stop162" /><stop
+   offset="1"
+   style="stop-color:#F2F7E8"
+   id="stop164" /><a:midPointStop
+   offset="0.0123"
+   style="stop-color:#FFFFFF" /><a:midPointStop
+   offset="0.6235"
+   style="stop-color:#FFFFFF" /><a:midPointStop
+   offset="1"
+   style="stop-color:#F2F7E8" /></radialGradient><circle
+   fill="url(#SVGID_9_)"
+   cx="232.459"
+   cy="219.704"
+   r="2.999"
+   id="circle166" /></g></g><g
+   id="g168"><image
+   overflow="visible"
+   opacity="0.25"
+   a:adobe-blending-mode="multiply"
+   a:adobe-opacity-share="1"
+   width="30"
+   height="30"
+   xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACEAAAAhCAYAAABX5MJvAAAACXBIWXMAAC4jAAAuIwF4pT92AAAA GXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAt9JREFUeNrsl91rE0EUxTOzm69a 05YmVWsUFdRKRfDBJ32Q+ocLRUTBBx+UYiOitlqRaEos2UT3w3P1jIzrbHZNVuiDAz9CNru5Z+69 M3O2Uvk/fg414zOCDzSxRwJiEJGkTBGKAatgEayCFr8rS4AwAp/BIRhTTDyPCDPzGoOfAhfANdAF zdT/SMBPoEf2wYBiQldmVAEBHjgJzoCL4DoFXAZroO4QMQRvwAvwDOyCtxQ3SWdF5QiQui+DDXAL 3ABXwDqvNygy3RPfwBHog1dgBzwBT8E7ENhCvJwMrIBNsAXugZvgPK83KdKzGlTzu8/fpWc64DSz +RV8oYhfTetPEdFiBu6SDc6+6lgRWZNo8P4Flk0zcMiMSGkSP+MPalQvJbhDASuplVB0RfkU0WUJ RuyNIcsWucoh15bYgFssQYfC1Bz7kcdsVNgrByzNHyJMFs4xA7fZA80CJcgb2uofycYHihlrh4hF ax9YL0mAGT7LKivsKmjLpLXjplVrH1ie0ryzlkWa9SxXnWR5QTv6ocUmWuMDqlLu8Di5ruk1Vzmq LEGdD5QtQln7yI8YespZUXbwzBi6cgzGsRTx14ZkxvFbDJeI9Laa/AMBgRUjTIsI6Yh69ANH0xzR DMMc8x/pMcTwjNIiIlqyHg1Jn8dvUmIWZGKv6THk/Jh4GWqFE3RTHW5auoQsTDj7B+Ah2JOzI8vU RNxQ2pYh8efYO4yHGNDubfNzkHWUJ9YqMf5yiZ7AmyEjtoAdCnjMjIj5TbycBhqxJxosT90SogoE j1kCI+A+Rbw0XmKaxzQlCXjz2GpOXUBITPFDztiUYJsNPzAC8kQklpBDrpQ+sxNZL0MR020wZlaC P2cTSvBHzMAg/f6hCu6qNfaGrJRLNCSbOS8/uyyBfL5n8JFrJy7a7Solpk1DYrynTvWBKcMBsxhw 9nEZL8S2GNtzuJo6YFOG1oor7a28iOdI8gLb47sAAwCDFN6m03jgxgAAAABJRU5ErkJggg=="
+   transform="matrix(0.24 0 0 0.24 243.7749 216.5547)"
+   id="image170"></image><g
+   id="g172"><radialGradient
+   id="SVGID_10_"
+   cx="247.374"
+   cy="219.7041"
+   r="2.9995"
+   gradientTransform="matrix(1 0 0 0.75 0 54.926)"
+   gradientUnits="userSpaceOnUse"><stop
+   offset="0.0123"
+   style="stop-color:#FFFFFF"
+   id="stop175" /><stop
+   offset="0.4235"
+   style="stop-color:#FAFCF6"
+   id="stop177" /><stop
+   offset="1"
+   style="stop-color:#F2F7E8"
+   id="stop179" /><a:midPointStop
+   offset="0.0123"
+   style="stop-color:#FFFFFF" /><a:midPoint
```

Comment:
i believe other browsers (at least firefox) have followed suit, so this might say "browser" instead of "chromium"